docs/builtins.md

```diff
@@ -1608,13 +1607,16 @@ Output
 
 #### Reference
 
-    hsl hue:num [saturation:num [lightness:num [alpha:num]]]
+    hsl:string hue:num [saturation:num [lightness:num [alpha:num]]]
+
+`hsl` returns a string to be used with the [`color`](#color) and
+[`clear`](#clear) functions. `hsl` takes one to four `num` arguments - `hue`,
+`saturation`, `lightness`, and `alpha` - and returns `string` containing a
+[CSS hsl function].
 
-The `hsl` function changes the color of the **stroke** and the **fill** to the
-given CSS hsl color. The `hue` value must be between 0 and 360. The
-`saturation`, `lightness` and `alpha` values must be between 0 and 100. See
-the Mozilla Developer documentation on the [CSS hsl function] for more
-details.
+The `hue` value must be between 0 and 360. The `saturation`, `lightness` and
+`alpha` values must be between 0 and 100. See the Mozilla Developer
+documentation on the [CSS hsl function] for more details.
 
 [CSS hsl function]: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl
```

Comment:
Interesting that this link was already here - didn't notice when I said to add "CSS" to the other one. But now it is duplicated. Can this one be removed?

Reply:
i updated this, not sure what happened...

Reply:
ah double trouble, i've got it twice now. Before i had CSS spelled out, because not everybody knows CSS and I spell it out on first use with any function doc....

Reply:
or so i tought.

Reply:
I did not notice you added `(Cascading Style Sheets)` to the link above. you can ignore this